src/include/gurt/common.h

```diff
@@ -73,6 +77,12 @@ int d_asprintf(char **strp, const char *fmt, ...);
 void *d_aligned_alloc(size_t alignment, size_t size);
 char *d_realpath(const char *path, char *resolved_path);
 
+#define D_SRAND(seedval)						\
+	d_srand48_r(seedval)
+
+#define D_RAND()							\
+	d_lrand48_r()
+
```

Comment:
[Question] Why macros? Wouldn't

    long int d_rand;
    void d_rand_seed(long int);

do?

[Nit] Might be better to place these line together with lines 66--69, to avoid disrupting the memory allocation stuff in the middle.

Reply:
@liw I was following what some of the other macros wrappers do honestly.  It looked like from what I saw most of the DAOS specific calls all use macros?

Reply:
My understanding is that those want to retain the functions, lines, and files of callers when printing debug/error messages.

Reply:
I believe so. It'd be wise to name them generically---not using "rand48"---though, for the implement could change in the future.

Reply:
@liw ok, updated. Thanks!